sdk/servicebus/azure-servicebus/azure/servicebus/_pyamqp/aio/_session_async.py

```diff
@@ -403,6 +403,7 @@ async def _wait_for_response(self, wait, end_state):
                 await self._connection.listen(wait=False)
 
     async def begin(self, wait=False):
+        _LOGGER.info("Session %r: Begin called", self.name, extra=self.network_trace_params)
```

Comment:
minor nit to minimize words in logging:
```suggestion
        _LOGGER.info("Session %r: Begin ", self.name, extra=self.network_trace_params)
```